### PR TITLE
Improve database semconv stability test formatting

### PR DESCRIPTION
**Pull request by trask**
Just a little renaming and static importing to improve the database semconv stability test formatting before I start adding a bunch more...

adding the script I used to static import so I can find and copy-paste-modify later when needed:

```
for file in $(find instrumentation -name '*Test.java'); do
  echo "Processing $file"

  # stable semconv

  negative_lookbehind='(?<!import static io.opentelemetry.instrumentation.testing.junit.db.)'

  if grep -Po "${negative_lookbehind}SemconvStabilityUtil.maybeStable" $file; then
    perl -i -pe "s/${negative_lookbehind}SemconvStabilityUtil.maybeStable/maybeStable/" $file
    sed -i "0,/^import/{s/^import/import static io.opentelemetry.instrumentation.testing.junit.db.SemconvStabilityUtil.maybeStable;\nimport/}" $file
  fi

  negative_lookbehind='(?<!import static io.opentelemetry.instrumentation.api.internal.)'

  if grep -Po "${negative_lookbehind}SemconvStability.emitStableDatabaseSemconv" $file; then
    perl -i -pe "s/${negative_lookbehind}SemconvStability.emitStableDatabaseSemconv/emitStableDatabaseSemconv/" $file
    sed -i "0,/^import/{s/^import/import static io.opentelemetry.instrumentation.api.internal.SemconvStability.emitStableDatabaseSemconv;\nimport/}" $file
  fi
done

./gradlew spotlessApply
```

### CODE REVIEW

File: testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/db/SemconvStabilityUtil.java [hunk 1]
```diff
@@ -39,14 +38,13 @@ private static Map<AttributeKey<?>, AttributeKey<?>> buildMap() {
     map.put(DB_SQL_TABLE, DB_COLLECTION_NAME);
     map.put(DB_CASSANDRA_TABLE, DB_COLLECTION_NAME);
     map.put(DB_MONGODB_COLLECTION, DB_COLLECTION_NAME);
-    map.put(DB_COSMOSDB_CONTAINER, DB_COLLECTION_NAME);
```
Comment (jaydeluca) [inline testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/db/SemconvStabilityUtil.java:42]: was this removal intentional?

Comment (trask) [reply testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/db/SemconvStabilityUtil.java:42]: oh yeah, I had meant to leave a github comment about it, it's unused (we don't have any cosmosdb tests)

**Review by jaydeluca — APPROVED** [review]
